pkg/controller/prpqr_reconciler/prpqr_reconciler.go

```diff
@@ -367,11 +457,6 @@ func generateProwjob(rc injectingResolverClient, defaulter periodicDefaulter, ba
 		Test: spec.Test,
 	}
```

Comment:
This is repeated here and in `resolveCiopConfig` - perhaps this needs to be also extracted and passed to both places as a parameter.